docs/.sphinx/_toc.yml.in

```diff
@@ -0,0 +1,167 @@
+                  - file: reference/gpu_libraries/fft
+                    subtrees:
+                    - entries: 
+                      - title: rocFFT
+                        url: https://rocmdocs.amd.com/projects/rocFFT/en/{branch}/
+                      - title: hipFFT
+                        url: https://rocmdocs.amd.com/projects/hipFFT/en/{branch}/
+                  - file: reference/gpu_libraries/rand
+                    subtrees:
+                    - entries: 
+                      - title: rocRAND
+                        url: https://rocmdocs.amd.com/projects/rocRAND/en/{branch}/
+                      - title: hipRAND
+                        url: https://rocmdocs.amd.com/projects/hipRAND/en/{branch}/
+                  - file: reference/gpu_libraries/solver
+                    subtrees:
+                    - entries: 
+                      - title: rocSOLVER
+                        url: https://rocmdocs.amd.com/projects/rocSOLVER/en/{branch}/
+                      - title: hipSOLVER
+                        url: https://rocmdocs.amd.com/projects/hipSOLVER/en/{branch}/
+                  - file: reference/gpu_libraries/sparse
+                    subtrees:
+                    - entries: 
+                      - title: rocSPARSE
+                        url: https://rocmdocs.amd.com/projects/rocSPARSE/en/{branch}/
+                      - title: hipSPARSE
+                        url: https://rocmdocs.amd.com/projects/hipSPARSE/en/{branch}/
+            - file: reference/gpu_libraries/c++_primitives
+              title: C++ Primitives
+              subtrees:
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/rocPRIM/en/{branch}/
+                    title: rocPRIM
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/hipCUB/en/{branch}/
+                    title: hipCUB 
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/rocThrust/en/{branch}/
+                    title: rocThrust
+            - file: reference/gpu_libraries/communication
+              title: Communication Libraries
+              subtrees:
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/RCCL/en/{branch}/
+                    title: RCCL
+            - url: https://rocmsoftwareplatform.github.io/MIOpen/doc/html/releasenotes.html
+              title: MIOpen - Machine Intelligence
+            - url: https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/
+              title: MIGraphX- Graph Optimization
+            - file: reference/computer_vision
+              subtrees:
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/MIVisionX/en/{branch}/
+                    title: MIVisionX
+                - entries:
+                  - url: https://rocmdocs.amd.com/projects/rocAL/en/{branch}/
+                    title: rocAL 
```

Comment:
Why is this file here? This seems to contain a lot of information not related to MIOpen.

Reply:
i think this is an accident. it will be fixed.

Reply:
Thanks for noting this. Fixed in [b42f3a6](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/b42f3a65c3a49164055aff69aea6635e37158429)